vendor/k8s.io/kubernetes/pkg/kubectl/cmd/get.go

```diff
@@ -213,6 +214,20 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 		Do()
 	err = r.Err()
 	if err != nil {
+		// providing an invalid cluster or context flag will yield
+		// an invalid configuration error when the client config
+		// fails to be obtained. If such error is received and either
+		// flag is set, such flag's value can then be blamed
+		if clientcmd.IsConfigurationMissing(err) {
+			context := cmdutil.GetFlagString(cmd, "context")
+			cluster := cmdutil.GetFlagString(cmd, "cluster")
```

Comment:
I would expect this to be handled by the code that merges in overrides from command line flags, not here